android/src/main/java/com/mapbox/reactnativemapboxgl/ReactNativeMapboxGLManager.java

```diff
@@ -45,9 +58,9 @@ public String getName() {
 
     @Override
     public MapView createViewInstance(ThemedReactContext context) {
-        MapView mv = new MapView(context, "pk.foo");
-        mv.onCreate(null);
-        return mv;
+        mapView = new MapView(context, "pk.foo");
+        mapView.onCreate(null);
```

Comment:
@bsudekum Looks like you may want to set a style here via `mapView.setStyle(Style.MAPBOX_STREETS)` for example.